### PR TITLE
Bear_Installation_Tool: Update to reflect the reworked tool

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: Developers/Bear_Installation_Tool.rst [hunk 1]
```diff
@@ -12,28 +12,59 @@ To install the tool, simply run:
...
+to upgrade all installed bears, or
+
+::
+
+    $ cib --upgrade CPPCheckBear PEP8Bear
+
+to upgrade the specified bears. However, if they are not installed, they will
+not be upgraded.
+
+cib also checks for bears' dependencies, using:
```
Comment (hypothesist) [inline Developers/Bear_Installation_Tool.rst]: double backticks

File: Developers/Bear_Installation_Tool.rst [hunk 2]
```diff
@@ -12,28 +12,59 @@ To install the tool, simply run:
...
-    $ coala-install
+    $ cib --install CPPCheckBear PEP8Bear
 
-and a question will pop-up. To install all the bears, simple type ``All``.
-If you want just a few bears, type ``Some``. Then, a list with all the
-bears, collected from PyPI, will pop up. You will now have to choose the bears
-you want to install, separated by commas, and along with the dependencies,
-the bears will install. To quit, simply type anything else.
+will install the specified bears only.
+``cib --uninstall`` works exactly the same way as ``cib -install``.
```
Comment (sils1297) [inline Developers/Bear_Installation_Tool.rst]: `-install` -> `--install`

**Comment by sils1297** [comment]
otherwise good

**Comment by Adrianzatreanu** [comment]
fixed, ack it :D

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/documentation/pull/93/commits/d491d700686ec68372c3f0d18c6a17202eb4fbb8

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**